includes/Modules/Subscribe_With_Google.php

```diff
@@ -47,7 +47,71 @@ public function register() {
 			return;
 		}
 
-		// TODO: Bring back SwG functionality after #3120 is merged.
+		// Register "access" meta field.
+		register_post_meta(
+			'',
+			'sitekit__reader_revenue__access',
```

Comment:
See above, this shouldn't use `sitekit_` as prefix. For settings and post meta, we typically use `googlesitekit_`, there is however a nuance to it:
* Site Kit allows to reset, wiping _basically_ everything related to Site Kit in the WordPress site.
* Resetting is generally regarded as a non-sensitive action. While we _do_ show a warning modal before resetting, Site Kit stores almost no data that wouldn't be easy to get back to after setting up the plugin again afterwards - that is because most of the _actual_ data comes from Google APIs.
* For configuration like this here though, it's different. If you configured 25 posts in a certain way for SwG and then reset Site Kit, you probably don't want to do that again, and you may not even remember how you configured them.
* For the above type of settings/meta, we have a special `googlesitekitpersistent_` prefix - any setting/meta that uses that won't get wiped on reset.

So for this meta field here, I'd suggest we use `googlesitekitpersistent_`.

Last but not least (nit-pick), let's only use single underscores to separate words.

Reply:
Sounds good, updated